sys-apps/bleachbit/bleachbit-1.19.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="
+	ar ast be bg bn bs ca cs da de el en_AU en_CA en_GB eo es et eu fa fi fo fr
+	gl he hi hr hu hy ia id it ja ko ku ky lt lv ms my nb nds nl nn pl pt pt_BR
+	ro ru se si sk sl sq sr sv ta te th tr ug uk uz vi zh_CN zh_TW"
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite(+)"
+
+inherit distutils-r1 eutils l10n
+
+DESCRIPTION="Clean junk to free disk space and to maintain privacy"
+HOMEPAGE="https://www.bleachbit.org"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gtk nls"
+
+RDEPEND="
+	dev-python/notify-python[$PYTHON_USEDEP]
+	gtk? ( dev-python/pygtk:2[$PYTHON_USEDEP] )"
+
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )"
+
+DOCS=( README.md )
+
+python_prepare_all() {
+	rem_locale() {
+		rm "po/${1}.po" || die "removing of ${1}.po failed"
```

Comment:
The `die` command won't expand variables, so it will literally print 'removing of ${1}.po failed'.

I'd use an if statement instead to evaluate `$?`, and use `eerror`, which will expand variables, followed by `die`.

Reply:
Thanks for the input. You mean something like implemented in the new commit?